pyproject.toml

```diff
@@ -29,11 +30,5 @@ test-command = [
     "whisper-cpp -h",
     "python -m whisper_cpp -h",
 ]
-
-[[tool.cibuildwheel.overrides]]
-select = "*-macosx_x86_64"
 environment.MACOSX_DEPLOYMENT_TARGET = "10.12"
 
-[[tool.cibuildwheel.overrides]]
-select = "*-macosx_arm64"
-environment.MACOSX_DEPLOYMENT_TARGET = "11.0"
```

Comment:
Why is this no longer necessary? (Can cibuildwheel validate the macOS compatibility?)

Reply:
scikit-build-core is smart enough to make sure the wheel is tagged as 11 on ARM even if MACOSX_DEPLOYMENT_TARGET is less than 11. Hatch doesn't support MACOSX_DEPLOYMENT_TARGET properly yet, but I'll try to add it soon (https://github.com/pypa/hatch/issues/1445).

I don't remember if it's in a released version, but yes: https://github.com/pypa/cibuildwheel/pull/1766 (delocate supports it, specifically)